docs/use/dapp-staking/for-devs/index.md

```diff
@@ -1,15 +1,98 @@
+
+:::important
+
+It's very important for dApp owners and their teams to get organized before and during the voting subperiod to market their products, run campaigns to attract as many stakers and tokens as possible during this period.
+
+:::
+
+**Build&Earn** is the subperiod when stakers and dApps start earning rewards;
+Users can still stake tokens during the Build&Earn subperiod to increase the rewards they get from staking. However, the amount staked during Build&Earn does not contribute to the Bonus Reward.
+
+At the end of a **Build&Earn** subperiod, the current period ends. A new period begins, and all tokens are `unstaked` from dApp(s) but remain `locked`. A new **Voting Subperiod** starts.
+
+As a dApp owner, you need to take the following parameters into consideration before using dApp Staking:
+
+### Parameters:
+
+|  | Shibuya | Shiden Network | Astar Network |
+| --- | --- | --- | --- |
+| Eras Per Period | 28 (~7days) | TBD | TBD |
+| Eras Per Voting Subperiod | 8 (~48hours) | TBD | TBD |
+| Eras Per Build&Earn Subperiod | 20 (~120hours) | TBD |TBD  |
+| Blocks Per Era | 1800 (~6hours) | TBD | TBD |
+| Unlocking Period | 4 Eras (~1 day) | 5 Eras (~5 days) | 10 Eras (~10 days) |
+
+### Tier System and Rewards
+
+dApp Staking introduces the concept of a tier system for dApps. It's important to fully understand the [tier mechanism](/docs/learn/dapp-staking/dapp-staking-protocol#tier-system) before proceeding with dApp Staking application and registration as a dApp owner.
+
+Currently, there are **4 tiers** with a limited number of slots per tier. Tier capacity for dApp staking is calculated at the start of each period based on the ASTR price as described [here](/docs/learn/dapp-staking/dapp-staking-protocol#tier-system).  
+
+The slots allocated to each tier can be viewed on the [dApp Staking Page](https://portal.astar.network/astar/dapp-staking/discover) of the Astar Portal.
+
+At the end of each **Build&Earn** subperiod, dApps are assigned to a tier based on the total value staked on them by users. Each tier has a **threshold** that a dApp must reach in order to access it.
+
+The threshold for tier 4 is fixed, but for the other tiers, the threshold is **dynamic** and calculated at the start of each new period based on the total number of slots for the period. To find out more, [click here](/docs/learn/dapp-staking/dapp-staking-protocol#tier-threshold-entry).
+
+**Rewards** for dApps are also **dynamic**, fluctuating from tier to tier. The higher the tier, the greater the number of ASTR tokens allocated from inflation as rewards for that tier.
+
+| Rewards allocated | Tier 1 | Tier 2 | Tier 3 | Tier 4 |
+| --- | --- | --- | --- | --- |
+| Astar Network | TBD | TBD | TBD | TBD |
+| Shiden Network | TBD | TBD | TBD | TBD |
+| Shibuya Network | TBD | TBD | TBD | TBD |
```

Comment:
This will be calculated dynamically so docs might not be the best place for such numbers. A link to the portal where reward pools are visible is preferable.

Reply:
I'm thinking of adding the rewards allocation in % for each tier here.

Reply:
Would that be informative for users?

I assume all they care about is concrete amount of ASTR/SDN they will get per era.